Makefile

```diff
@@ -157,12 +157,17 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+
+.PHONY: markdownlint
+markdownlint: ## Check formatting of documentatio n sources
+	@$(DOCKERCMD)	run --platform linux/amd64 -v .\:/workdir\:z docker.io/davidanson/markdownlint-cli2:v0.17.1  markdownlint-cli2 "Documentation/**.md" "\#Documentation/Helm-Charts/**" --config .markdownlint-cli2.cjs
```

Comment:
I'm willing to use `markdownlint-cli2` on the system for this -- as we did with pylint -- but I don't want to have to pull a docker container on mac, then wait eons to load the repo into the container context, just to run a linter.

Reply:
I was clear the first time, I don't want to use the docker container. Sorry

Reply:
@BlaineEXE  wrote:
> I was clear the first time, I don't want to use the docker container. Sorry

got it now. I will rework this to have the make target rely on a locally installed `markdownlint-cli2`. 

And based on the other comments, I will remove the change to the workflow file.